src/schema/v2/fields/money.ts

```diff
@@ -50,11 +51,18 @@ export const amount = (centsResolver) => ({
       type: GraphQLString,
       defaultValue: ",",
     },
+    disambiguate: {
+      type: GraphQLBoolean,
+      defaultValue: false,
+    },
   },
   resolve: (obj, options) => {
     const cents = centsResolver(obj)
+
     const symbol =
-      options.symbol || obj.symbol || symbolFromCurrencyCode(obj.currencyCode)
+      options.symbol ||
+      obj.symbol ||
+      symbolFromCurrencyCode(obj.currencyCode, options.disambiguate)
```

Comment:
I did some digging to learn why we'd need `obj.symbol` since it's redundant with `obj.currencyCode` and also curious what objects expose a symbol in addition to the currency code. It turned out it's introduced in https://github.com/artsy/metaphysics/pull/181 for [sale artworks](https://github.com/artsy/gravity/blob/ef9b996155ccb9f6adeffddaff03213f8e96c3e8/app/models/domain/sale_artwork.rb#L142), and we just used the symbol, not the currency code, for formatting at that point.

_I think_ eventually we want to use the currency code on the object instead of the symbol (since it can be ambiguous). For now I'd suggest we keep the behavior and update the description of the `disambiguate` argument to make it clear that it'll only have effect when there is no symbol explicitly passed in as an argument or symbol from the object itself. We can separately investigate if it's ok to ignore the symbol from the object.

In our use cases, Exchange just exposes the [currency code](https://github.com/artsy/exchange/blob/2b581ebbd9d4d2d1af3327ae546af89f1ea8ce08/app/graphql/types/order_interface.rb#L18) and not the symbol, so I think the `disambiguate` here would work.